projects/origin-release/golang-1.13/golang.spec

```diff
@@ -0,0 +1,21 @@
+Name:		golang
+Version:	v1.13.0
```

Comment:
1.13.0 ? PR says `1.13.1`

Reply:
Compare with the 1.12 file. This is for building fake RPMs only and it has been kept at 1.12.0 as well although the version is actually at 1.12.9 now.

Reply:
ah... does not make sense, but ok :-)